manager/integration/tests/common.py

```diff
@@ -4039,6 +4080,27 @@ def backing_image_feature_supported(client):
         return False
 
 
+def cleanup_all_recurring_jobs(client):
+    recurring_jobs = client.list_recurring_job()
+    for recurring_job in recurring_jobs:
+        try:
+            client.delete(recurring_job)
+        except Exception as e:
+            print("\nException when cleanup recurring job ", recurring_job)
+            print(e)
```

Comment:
Minor, but u can merge two lines as one print statement.

Reply:
Without going over the line character's limit, I think the error would be easier to read on a separate line. And this is currently consistent with the others.

Reply:
I mean `print("....", var, e)` or even use .format. But that's fine if most of the code has been done with this style already.